src/Ant/Modal.elm

```diff
@@ -273,6 +279,16 @@ withMask toggle (Modal opts contents) =
     Modal { opts | showMask = toggle } contents
 
 
+{-| Specify a percentage value for positioning the modal vertically with respect to the documents height.
+
+By default this value is set to `15`.
+
+-}
+withVerticalOffset : Float -> Modal msg -> Modal msg
```

Comment:
I think this function should allow for either percentage or pixel-based offsets. For example, in the case of the parlez-vous embed, the document height changes based on the number of comments rendered. So having a percentage based offset would mean that the modal would render at unpredictable offsets on various comment tree configurations.

Reply:
good idea, i have made two functions for this to keep the api simpler and avoid having to expose the union type